kubetools/kubernetes/config/container.py

```diff
@@ -128,6 +128,9 @@ def make_container_config(
                 'readonly': True,
             })
 
+    # Remove annotations from container data
+    if 'annotations' in container:
+        container.pop('annotations')
```

Comment:
I had to add this otherwise the annotations was also added to the container definition, see the last 3 lines of the example  below:
```
apiVersion: apps/v1
kind: Deployment
metadata:
  annotations: {
    app.kubernetes.io/managed-by: kubetools,
    app.kubernetes.io/name: generic-app-with-annotations
  }
  labels: {
    kubetools/name: generic-app-with-annotations,
    kubetools/project_name: generic-app,
    kubetools/role: app
  }
  name: generic-app-with-annotations
spec:
  replicas: 1
  revisionHistoryLimit: 5
  selector:
    matchLabels: {
      kubetools/name: generic-app-with-annotations,
      kubetools/project_name: generic-app,
      kubetools/role: app
    }
  template:
    metadata:
      labels: {
        kubetools/name: generic-app-with-annotations,
        kubetools/project_name: generic-app,
        kubetools/role: app
      }
    spec:
      containers:
      - command: [generic-command]
        containerContext: generic-context
        env:
        - {name: KUBE, value: 'true'}
        image: generic-image
        imagePullPolicy: Always
        livenessProbe:
          httpGet: {path: /ping, port: 80}
          timeoutSeconds: 5
        name: webserver
        readinessProbe:
          httpGet: {path: /ping, port: 80}
          timeoutSeconds: 5
        annotations: {
          app.kubernetes.io/name: generic-app-with-annotations
         }
```

Reply:
This is the consequence of the problem I'm referring to in another comment

Reply:
It has been removed in this commit https://github.com/EDITD/kubetools/pull/165/commits/12c0b0863d2a93f7544afcaf3288c7c82db5d899